timestamp-client/src/main/java/com/palantir/timestamp/RateLimitedTimestampService.java

```diff
@@ -40,7 +40,7 @@
  */
 @ThreadSafe
 public class RateLimitedTimestampService implements TimestampService {
```

Comment:
`TransactionManagers` is in `atlasdb-config`, which doesn't bring in `timestamp-impl` (and for cleanliness I'd rather it didn't - that project shouldn't need to know about the guts of the persistent timestamp service)